galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager/src/main/java/dev/galasa/galasaecosystem/internal/properties/DockerRegistry.java

```diff
@@ -21,12 +21,12 @@
  * 
  * @galasa.required No
  * 
- * @galasa.default docker.galasa.dev
+ * @galasa.default harbor.galasa.dev/docker_proxy_cache
  * 
  * @galasa.valid_values a valid hostname
  * 
  * @galasa.examples 
- * <code>galasaecosystem.docker.version=docker.galasa.dev</code>
+ * <code>galasaecosystem.docker.version=harbor.galasa.dev/docker_proxy_cache</code>
```

Comment:
Hi Fiona, thanks for making these changes. Thinking about this - while harbor.galasa.dev/docker_proxy_cache is correct for us and the registry we use, a Galasa user might not use Harbor and instead might use their own registry or even Dockerhub. We should replace this with a more generic example to just demonstrate this needs to be a valid hostname of a Docker registry.
Therefore, I'd change the value to just be 'docker.io'.

Reply:
That makes sense, will update, thanks